pkg/mutation/stats_reporter_test.go

```diff
@@ -0,0 +1,85 @@
+package mutation
+
+import (
+	"testing"
+
+	"go.opencensus.io/stats/view"
+)
+
+func TestReportIterationConvergence(t *testing.T) {
+	r, err := NewStatsReporter()
+	if err != nil {
+		t.Errorf("newStatsReporter() error %v", err)
```

Comment:
t.Fatal - otherwise calling methods on `r` will panic

Reply:
I changed the NewStatsReporter function signature to not return an error.  I did so by not using `tag.new()` and returning a raw `context.Background()